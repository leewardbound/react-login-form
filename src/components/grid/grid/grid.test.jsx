import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Grid from '.';

const renderer = new ShallowRenderer();

describe('Grid', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <Grid>
        <h1>Lorem ipsum</h1>
      </Grid>
    );

    expect(component).toMatchSnapshot();
  });
});
