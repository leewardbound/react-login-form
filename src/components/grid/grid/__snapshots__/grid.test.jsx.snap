// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render correctly 1`] = `
<div
  className="fZysxz"
>
  <h1>
    Lorem ipsum
  </h1>
</div>
`;
